### DISCUSSION TITLE
[x][closed][blueprint] Current BLUEPRINT metadata does not contain enough information to discern between tracks

### DISCUSSION THREAD

**Discussion by sveinugu** [fairtracks/fairtracks_standard#102] [Team Posts]
@dzerbino 

See the following GSuite results:

https://hyperbrowser.uio.no/temp1/u/sveinugu/h/trackfind-gsuite-examples

Metadata for many of the tracks are almost exactly the same, meaning that we should have probably included a bit more info to be able to discern between them.

**Comment by dzerbino** [comment]
Sorry, I don't understand what this report is telling me, could you please clarify?

**Comment by dzerbino** [comment]
Could you please double check this issue? Looking at the Blueprint file I sent you June 10th (i.e. before the last edit to this ticket) the long label for track S0138VH1.ERX1007381.H3K4me1.bwa.GRCh38.20151029.bw is "H3K4me1 on naive B cell in tonsil from donor T14_11" but according to the report linked in the ticket the label_long attribute is "H3K4me1 on naive B cell". Either this is an old issue which was corrected at some point, or there is a problem with the parsers.

**Comment by sveinugu** [comment]
So this issue was stupidly copied from the minutes from the meeting in Lisbon. I did not check whether it had been fixed. So it seems you have fixed it. I will redo the example once everything is in place and see whether the issue has been handled well enough.

**Comment by dzerbino** [comment]
Well, at least it's fixed ;)

**Comment by sveinugu** [comment]
@dzerbino Still some duplicate 'label_long' values. For instance local_ids:

C000S5B1.minusStrandMulti.star_grape2_crg.GRCh38.20150815.bw
C000S5B1.ipsa_junctions.ipsa_grape2_crg.GRCh38.20150817.gff
C000S5B1.plusStrandMulti.star_grape2_crg.GRCh38.20150815.bw
C000S5B1.splicing_ratios.ipsa_grape2_crg.GRCh38.20150817.gff
C000S5B1.minusStrand.star_grape2_crg.GRCh38.20150815.bw
C000S5B1.plusStrand.star_grape2_crg.GRCh38.20150815.bw

have all the same label_long value: total-RNA-Seq on CD14-positive, CD16-negative classical monocyte in venous blood from donor C000S5

Similarly:

C000S5H2.ERX173534.H3K4me1.bwa.GRCh38.broad.20150527.bb and C000S5H2.ERX173534.H3K4me1.bwa.GRCh38.20150528.bw are both "H3K4me1 on CD14-positive, CD16-negative classical monocyte in venous blood from donor C000S5"

It actually seems that adding a properly detailed content_type attribute will solve all the remaining duplications, but it seems we will be able to be detailed enough for at least the star file subtypes (above).

**Comment by dzerbino** [comment]
Now corrected, new JSON sent by e-mail

**Comment by sveinugu** [comment]
Very nice!! Closing
